nselib/rdp.lua

```diff
@@ -86,6 +102,37 @@ Packet = {
 
   },
 
+  ConfCreateResponse = {
+
+
+    new = function(self, data)
+      local o =  {}
+      setmetatable(o, self)
+      self.__index = self
+      return o
+    end,
+
+    parse = function(data)
+      local ccr = Packet.ConfCreateResponse:new()
+      local pos = 67
+
+      while data:len() > pos do
+        block_type, block_len  = string.unpack("I2I2", data, pos)
+        if block_type == 0x0c01 then
+          proto_ver = string.unpack("I4", data, pos + 4)
```

Comment:
Please use explicit endianness here (probably little-endian, `"<I4"`) and elsewhere with integer types so that the code runs the same on big-endian systems.

Reply:
Thanks for the feedback @dmiller-nmap   I'll address and retest. If everything works correctly I'll commit this evening.